lib/api/graph.js

```diff
@@ -43,8 +44,63 @@ function snapshot(query, from, to, eventQuery, callback){
     client.request("GET", "/graph/snapshot", params, callback);
 }
 
+/*section: graph
+ *comment: create an embed graph of a metric query
+ *params:
+ *  graph_json: The request array to pass create in the embed
+ *  timeframe: (1_hour, 4_hours, 1_day, 2_days, and 1_week)
+ *  size: (small, medium, large, xlarge)
+ *  legend: yes or no
+ *  title: The title of the embed
+ *  callback: function(err, res)
+ *example: |
+ *  ```javascript
+ *  var dogapi = require("dogapi");
+ *  var options = {
+ *    api_key: "api_key",
+ *    app_key: "app_key"
+ *  };
+ *  dogapi.initialize(options);
+ *  var query = "system.cpu.idle{*}";
```

Comment:
`query` isn't used, you could use it below as `"q": query,`, or remove, either is fine.